### DISCUSSION TITLE
grid is shaking.

### DISCUSSION THREAD

**Discussion by seonhyePark** [Comcast/react-data-grid#2976] [Q&A]
React-data-grid 7.0.0-beta.11 is in use.
When zooming in and out of the screen, the grid shakes.
when I look at devtools,
-rdg-grid-inline-size and rdg-grid-block-size  changes between the two values.
For your information
The width size of the column is set to %. 
The test environment is 1920*1080, 1366*768 resolution

 <div style={{width:"98%",minWidth: "1050px", height: "92%"}}>
          <div style={{width:"100%", height:"96%",display:"flex",flexDirection:"row",justifyContent: "space-between", alignItems: "flex-start"}}>
                    <div style={{ width: detailInfo ? "55%" : "100%", height:"98%"}}>
                        <DataGrid columns={chargerColumns} rows={chargersInfo}  headerRowHeight={24} rowHeight={30} onScroll={onScroll} onCopy={onCopy} rowClass={(row) => (detailInfo && detailInfo.id === row.id ? 'selected-row' : 'normal-row')} sortColumns={sortColumns} onSortColumnsChange={(sortColumns)=>onSortColumnsChange(sortColumns)}
                            style={{width: "99%", height: "100%"}}/>
                    </div>
</div>
</div>

**Comment by vijay-t13** [comment]
Any Update on this ??
